examples/pytorch/image-classification/run_image_classification_no_trainer.py

```diff
@@ -213,7 +213,13 @@ def main():
     # If we're using tracking, we also need to initialize it here and it will by default pick up all supported trackers
     # in the environment
     accelerator = (
-        Accelerator(log_with=args.report_to, logging_dir=args.output_dir) if args.with_tracking else Accelerator()
+        Accelerator(
+            log_with=args.report_to,
+            logging_dir=args.output_dir,
+            gradient_accumulation_steps=args.gradient_accumulation_steps,
+        )
+        if args.with_tracking
+        else Accelerator(gradient_accumulation_steps=args.gradient_accumulation_steps)
```

Comment:
Maybe define `log_kwargs` in an if/else then do just one call to `Accelerator`?

Reply:
I tried doing what you describe. Let me know if this is correct. :)